script/openqa-validate-yaml

```diff
@@ -19,6 +19,52 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+=head1 NAME
```

Comment:
I must say, I'm not a fan of repeating documentation.
This could get outdated when options are added or changed.
There's https://metacpan.org/pod/Getopt::Long::DescriptivePod which can generate pod from the Getopt::Long::Descriptive options.
It could be called during installation, or alternatively, by the users doing changes to the script.

Other opinions?

Reply:
I completely agree -- if there's a good way to generate the same thing to keep things in sync that's much better.